### ISSUE TITLE
Update picture element to allow aria-hidden

### ISSUE THREAD

**Issue by scottaohara** [IBMa/equal-access#503]
While we do not think it's a good idea to allow additional roles / aria-* attributes on the picture element, since it is not supposed to be directly exposed to the a11y-api, there are potential use cases for setting the entire picture, and thus the graphics it represents as hidden. While this should be done with alt="" on the fallback image element, allowing aria-hidden=true on the picture itself should be allowed as this will also allow authors to hide the element.

Action item:
The aria-hidden attribute can be specified on the picture element as the only allowed aria attribute.

**Comment by tombrunet** [comment]
Need to come up with a testcase on this one. Not sure if it's already been fixed with the other ARIA updates.

**Comment by scottaohara** [comment]
@tombrunet in case this is helpful: https://w3c.github.io/html-aria/tests/aria-hidden-picture.html

**Comment by carmacleod** [comment]
@tombrunet It hasn't been fixed yet, but I'll take this. I have a few similar ones lined up (i.e. elements with additional attributes allowed that don't relate to implicit role or there is no implicit role), but the PR isn't ready yet - needs cleanup and test cases.

**Comment by shunguoy** [comment]
@carmacleod  @tombrunet  the picture element is merely a container for switching src for a single image. The picture element itself doesn't display anything and doesn't have any semantic meaning (except it's so named), and probably that's the reason why it has "No role or aria-* attributes" allowed in the Aria in HTML recommendation.  A user can just simply use alt or role on the img element to make it accessible or inaccessible.  What's the advantage to use an aria-hidden='true' in a picture element?  
    The aria-hidden='true' should be used with great caution for a container element with real content as children.  Consider the following case:
      ```
<picture aria-hidden='true'>
         <source srcset='xxxxxx' ...>
         <source srcset='yyyyyy' ...>
         <img src='zzzzzz' title='my content image' />
     </picture>
```
    The use of the aria-hidden='true' in the picture element alone is not enough, and the use of alt='' or role='presentation | none' to replace the title is also needed for supporting users who use audio interfaces. Therefore, It's easier and more straightforward to just use alt or role in the img element. 
    It would be more helpful If you can provide a more practical use case for this. Otherwise, I would suggest following the Aria recommendation, and closing this issue.

**Comment by carmacleod** [comment]
@shunguoy Please see the latest version of the ARIA in HTML spec for picture: https://w3c.github.io/html-aria/#el-picture
It has been updated to say:
> Authors MAY specify the aria-hidden attribute on the picture element. Otherwise, no other allowed aria-* attributes.

This spec is moving to a more "evergreen" model, where the latest is the recommendation. To ensure that the spec remains valid, changes are only pulled in when there is agreement/implementation from at least 2 conformance checkers. For this particular change, axe and arc toolkit have both implemented. So Checker needs to implement it too, or be noted as nonconforming. :)

In Scott's example: https://w3c.github.io/html-aria/tests/aria-hidden-picture.html an img is contained within a picture. Checker's guidance could say that using `alt=""` or `aria-hidden=true` on the `img` is the preferred option. But since there is nothing invalid or detrimental to accessibility if `aria-hidden=true` is used on the `picture`, we shouldn't flag it as a violation.

**Comment by shunguoy** [comment]
add more estimated days to cover all other similar cases from the latest aria in HTML draft
